packages/discord-player/src/Player.ts

```diff
@@ -104,6 +106,19 @@ export class Player extends PlayerEventsEmitter<PlayerEvents> {
 
         super([PlayerEvent.Error]);
 
+        if(options.ffmpegPath) {
+            if(typeof options.ffmpegPath !== "string") throw new TypeError(`Expected type "string" for options.ffmpegPath. Got ${typeof options.ffmpegPath} instead`)
+            if(!existsSync(options.ffmpegPath)) throw new Error(`The pathway provided for options.ffmpegPath does not exists.`)
+
+            try {
+                spawnSync(options.ffmpegPath, ['--help'])
```

Comment:
lets not do this here

Reply:
ffmpeg is lazy loaded

Reply:
Do we wait for it to fail on when ffmpeg is actually called?

Reply:
Yes. Because discord-player also includes `skipFFmpeg` mode, where ffmpeg will not be used when possible.